boofuzz/__init__.py

```diff
@@ -591,7 +590,7 @@ def s_binary(value, name=None):
         pair = parsed[:2]
         parsed = parsed[2:]
 
-        value += six.int2byte(int(pair, 16))
+        value += int(pair, 16).to_bytes(1, "little")
```

Comment:
Since six defines
```python
int2byte = struct.Struct(">B").pack
```
it would be more accurate to replace it with `to_bytes(1, "big")`, wouldn't it?
https://docs.python.org/3/library/struct.html#byte-order-size-and-alignment

Reply:
The size or length is 1 so it doesn't really matter what endianness we set. Still, it would be more accurate.

Reply:
Alright, I agree